docs/source/trainer.rst

```diff
@@ -580,9 +580,17 @@ You can override the default behavior by initializing the :class:`~pytorch_light
 callback, and adding it to the :paramref:`~pytorch_lightning.trainer.trainer.Trainer.callbacks` list.
 See :ref:`Saving and Loading Weights <weights_loading>` for how to customize checkpointing.
 
+.. code-block:: python
```

Comment:
```suggestion
.. testcode::
```

Reply:
why?

Reply:
this should try to execute the test, which will be testing that the docs are aligned with codebase :]